src/TreeView/TreeView.tsx

```diff
@@ -525,7 +532,7 @@ const SubTree: React.FC<TreeViewSubTreeProps> = ({count, state, children}) => {
 
   // Announce when content has loaded
   React.useEffect(() => {
-    if (state === 'done') {
+    if (isPending && state === 'done') {
```

Comment:
As discussed in our zoom call: Could you add a small code comment explaining why `isPending` is necessary?